src/main/app/src/app/klanten/persoonsgegevens/persoonsgegevens.component.ts

```diff
@@ -0,0 +1,40 @@
+import {AfterViewInit, Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {KlantenService} from '../klanten.service';
+import {SessionStorageService} from '../../shared/storage/session-storage.service';
+import {Persoon} from '../model/personen/persoon';
+
+@Component({
+    selector: 'zac-persoongegevens',
+    templateUrl: './persoonsgegevens.component.html'
+})
+export class PersoonsgegevensComponent implements OnInit, AfterViewInit {
+
+    @Input() bsn: string;
+    @Output() delete = new EventEmitter<Persoon>();
+
+    persoon: Persoon;
+    klantExpanded: boolean = sessionStorage.getItem('klantExpanded') === 'true';
```

Comment:
Dit lijkt me niet nodig, omdat hij in de oninit al geinitialiseerd wordt. Beter is het om gebruik te maken van de sessionstorage service, voor het geval dat we in de toekomst daar nog iets van een medewerker id aan moeten koppelen